code/game/objects/structures/flora.dm

```diff
@@ -193,7 +193,7 @@
 	//If it *is* an item stack, we don't want to go through 50 different iterations of a new object where it just gets qdeleted after the first
 	. = list()
 	for(var/product in products_to_create)
-		var/amount_to_create = products_to_create[product]
+		var/amount_to_create = products_to_create[product]*product_amount_multiplier
```

Comment:
round this?

Reply:
?

Reply:
```suggestion
		var/amount_to_create = round(products_to_create[product]*product_amount_multiplier, 1)
```

So you're not creating 1.2 products or something

Reply:
okay